settings/ChangePassword/ChangePassword.js

```diff
@@ -205,6 +213,16 @@ class ChangePassword extends Component {
               </div>
             </Col>
           </Row>
+          <Row>
+            <Col xs={6}>
+              <Field
+                name="captcharesponse"
+                component={Recaptcha}
+                refName={this.captchaRef}
+                siteKey="6LdpJmkUAAAAACPqv1O2YJRHsNoaLSKzjAxPZmkM"
```

Comment:
As you are making a custom component with a RECAPTCHA, you can move siteKey out of a template.

Reply:
Of course.